apollo-spaceport/Cargo.toml

```diff
@@ -10,15 +10,15 @@ publish = false
 
 [dependencies]
 bytes = "1.1.0"
-clap = { version = "3.1.3", default-features = false, features = ["derive"] }
+clap = { version = "3.1.3", default-features = false, features = ["std", "derive"] }
 flate2 = "1.0.22"
 prost = "0.9.0"
 prost-types = "0.9.0"
 reqwest = { version = "0.11.10", default_features = false, features = [
     "rustls-tls",
     "json",
 ] }
-serde = "1.0.136"
+serde = {version = "1.0.136", features = ["derive"] }
```

Comment:
Can't see in your changes why we need feature `derive`

Reply:
when building spaceport independently of the rest, compilation fails in files generated by tonic:
```
error[E0433]: failed to resolve: could not find `Serialize` in `serde`
   --> /home/geal/dev/router/target/debug/build/apollo-spaceport-794c79ea9020bef4/out/report.rs:369:17
    |
369 | #[derive(serde::Serialize, Clone, PartialEq, ::prost::Message)]
    |                 ^^^^^^^^^ could not find `Serialize` in `serde`

error[E0433]: failed to resolve: could not find `Serialize` in `serde`
   --> /home/geal/dev/router/target/debug/build/apollo-spaceport-794c79ea9020bef4/out/report.rs:378:17
    |
378 | #[derive(serde::Serialize, Clone, PartialEq, ::prost::Message)]
    |                 ^^^^^^^^^ could not find `Serialize` in `serde`
```

We usually do not see this error because the derive feature is added in lots of other places in the router and its dependencies

Reply:
so it's not due to changes in this PR, just a comppilation failure we discovered while looking into it

Reply:
Okidok